ui/page/root/wallet_list.go

```diff
@@ -351,7 +351,7 @@ func (pg *WalletSelectorPage) listenForSyncProgressNotifications() {
 	for _, w := range allWallets {
 		err := w.AddSyncProgressListener(syncListener, WalletSelectorPageID)
 		if err != nil {
-			log.Errorf("Error adding sync progress listener: %v", err)
+			log.Errorf("3. Error adding sync progress listener: %v", err)
```

Comment:
We can use the method receiver's name as a prefix so we don't have to number everywhere, or was this just for debugging?

Reply:
Debugging. Will remove.